Readme.md

```diff
@@ -1,24 +1,20 @@
-# WebMemex/memextension
+# WorldBrain's WebMemex
```

Comment:
Oh, now I understand entirely! 
That title was not meant to be for your project. It was meant for the WorldBrain version of the WebMemex. 
Sorry for the confusion :)

Failed to exclude the Readme file from the PR to the Webmemex repo!